vulnerabilities/importers/pypa.py

```diff
@@ -28,16 +29,21 @@ class PyPaImporter(Importer):
     importer_name = "Pypa Importer"
 
     def advisory_data(self) -> Iterable[AdvisoryData]:
+        progress_bar_for_package_fetch = ChargingBar("\tFetching Packages")
```

Comment:
![Screenshot from 2024-01-17 18-07-26](https://github.com/nexB/vulnerablecode/assets/29133904/4b8ea973-5b8f-4ac5-a37e-9df7184a0b6a)
I think you should add a new line before or after printing the `Fetching Packages` bar

Reply:
Adding a new line won't work because as the `progress` library uses `\r` carriage return under the hood !